src/index.html

```diff
@@ -0,0 +1,74 @@
+                  -25 244 -86 59 -59 78 -125 71 -253 -11 -221 -74 -307 -659 -897 l-338 -340 0
+                  -127 0 -127 710 0 710 0 0 145 0 145 -495 0 c-318 0 -495 4 -495 10 0 5 157
+                  166 349 357 298 298 358 362 414 448 110 168 147 281 147 449 0 175 -39 275
+                  -150 386 -117 118 -250 168 -460 175 -69 2 -154 -1 -190 -6z" />
+              <path d="M1390 7295 l0 -1985 296 0 295 0 -4 1393 c-3 781 -9 1453 -15 1532
+                  -23 322 -25 398 -12 393 7 -2 270 -750 584 -1661 l571 -1657 286 0 286 0 10
+                  23 c5 12 272 757 594 1654 359 1004 589 1633 597 1633 9 0 11 -21 7 -92 -3
+                  -51 -11 -176 -17 -278 -6 -107 -12 -766 -15 -1562 l-5 -1378 311 0 311 0 0
+                  1985 0 1985 -456 -2 -456 -3 -566 -1579 c-312 -869 -572 -1581 -577 -1583 -6
+                  -1 -257 710 -560 1582 l-550 1585 -457 0 -458 0 0 -1985z" />
+              <path d="M1390 2565 l0 -1987 703 5 c756 4 781 6 1056 63 255 52 514 158 716
+                   291 139 92 327 282 424 426 154 232 247 490 297 822 24 158 30 560 10 728 -56
+                   483 -206 818 -495 1108 -278 278 -619 434 -1101 505 -98 15 -227 18 -862 21
+                   l-748 4 0 -1986z m1535 1422 c290 -52 534 -175 689 -348 166 -185 267 -436
+                   307 -764 17 -140 14 -464 -5 -607 -76 -560 -340 -906 -808 -1059 -194 -63
+                   -503 -93 -888 -87 l-175 3 -3 1443 -2 1444 397 -5 c308 -3 419 -8 488 -20z" />
+              <path d="M5780 2566 l0 -1986 768 0 c436 0 814 4 877 10 811 74 1253 534 1195
+                   1244 -17 203 -52 319 -138 451 -112 174 -297 292 -558 356 -86 21 -113 33
+                   -102 44 2 3 40 14 84 26 43 11 122 41 174 66 80 40 109 61 186 137 79 79 98
+                   105 143 196 81 165 94 229 95 455 0 174 -2 198 -26 287 -66 243 -209 410 -443
+                   519 -154 71 -353 122 -607 155 -101 14 -258 17 -885 21 l-763 5 0 -1986z
+                   m1456 1433 c294 -30 473 -119 552 -275 38 -76 52 -144 52 -252 -1 -255 -114
+                   -423 -334 -497 -141 -47 -235 -55 -673 -55 l-403 0 0 545 0 545 348 0 c192 0
+                   398 -5 458 -11z m69 -1619 c358 -41 554 -177 621 -431 22 -85 22 -261 0 -349
+                   -42 -166 -145 -305 -276 -374 -161 -85 -379 -109 -925 -104 l-290 3 -3 633 -2
+                   632 392 0 c216 0 434 -5 483 -10z" />
+          </g>
+      </svg>
+      <h1 class="text-white text-lg ml-2">Microverse Movie DataBase</h1>
+  </div>
+  <nav class="space-x-4">
+      <a href="#" class="text-white">Movies</a>
+      <a href="#" class="text-white">Actors</a>
+      <a href="#" class="text-white">Directors</a>
+  </nav>
+</header>
+
+
+
+<section id="main-section" class="container mx-auto px-4 py-8">
+  <div class="grid grid-cols-3 gap-4 justify-center">
+    
+  </div>
+</section>
+
+
+
+<footer class="fixed bottom-0 left-0 right-0 py-2 text-center bg-gray-800 text-white">
+  <p>Created by Microverse under CC license</p>
```

Comment:
[OPTIONAL] For this project, we do not use a CC license so we do not need to give any credit for it, but you can keep it there if you want:)